### PR TITLE
make changes to getting started and setup

### PR DESCRIPTION
**Pull request by gomathyk**
## Purpose
make changes to getting started and setup

### CODE REVIEW

File: en/docs/getting-started/overview.md [hunk 1]
```diff
@@ -87,8 +87,8 @@ efficiency.
 
 WSO2 Identity Server has the ‘ [Jaggery](http://jaggeryjs.org/) ’ user
 interface for end users. Apart from the [Management
-Console](/installation-guide/getting-started-with-the-management-console) , an [end user
-view](/using-wso2-identity-server/using-the-end-user-dashboard) is available to manage profiles,
+Console](../../installation-guide/getting-started-with-the-management-console) , an [end user
```
Comment (sherenewso2) [inline en/docs/getting-started/overview.md]: there is no "installation-guide" folder anymore..let's change this link to "../../setup/getting-started-with-the-management-console"

File: en/docs/getting-started/provisioning-architecture.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ and Group resources in a RESTful way.
 The following is a high level overview of SCIM Service Provider
 architecture of IS.
 
-![scim-service-provider-architecture](/assets/img/introduction/scim-service-provider-architecture.png)
+![scim-service-provider-architecture](../../assets/img/introduction/scim-service-provider-architecture.png)
```
Comment (sherenewso2) [inline en/docs/getting-started/provisioning-architecture.md]: there is no "introduction" sub folder anymore. let's change this link to "../../assets/img/**getting-started**/scim-service-provider-architecture.png". 

please change in all relevant places

File: en/docs/getting-started/single-sign-on-and-identity-federation.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ To understand how single sign-on works, it is useful to first examine
 how authentication works in a non SSO environment. The following diagram
 depicts this flow.
 
-![Authentication in a non SSO environment](/assets/img/concepts/authentication-in-non-sso.png)
+![Authentication in a non SSO environment](../../assets/img/concepts/authentication-in-non-sso.png)
```
Comment (sherenewso2) [inline en/docs/getting-started/single-sign-on-and-identity-federation.md]: no more "concepts" folder. let's change it to "../../assets/img/getting-started/authentication-in-non-sso.png". please change in all relevant places

File: en/docs/tutorials/configuring-a-service-provider-for-adaptive-authentication.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ Follow the steps below to set up the sample application to try out
 adaptive authentication using a sample application.
 
 1.  PickUp sample web application (SAML).
-    1.  Download the [saml2-web-app-pickup-dispatch.com.war](/assets/attachments/saml2-web-app-pickup-dispatch.com.war)
+    1.  Download the [saml2-web-app-pickup-dispatch.com.war](../../assets/attachments/saml2-web-app-pickup-dispatch.com.war)
```
Comment (sherenewso2) [inline en/docs/tutorials/configuring-a-service-provider-for-adaptive-authentication.md]: this .war file and the attachments folder has not been added to the repo. please check.

File: en/docs/tutorials/configuring-a-service-provider-for-adaptive-authentication.md [hunk 5]
```diff
@@ -53,15 +53,15 @@ adaptive authentication using a sample application.
...
 
 2.  Deploy the sample authenticator dependency and web application in
     WSO2 IS.
 
-    1.  Download the [org.wso2.carbon.identity.sample.extension.authenticators-5.7.0.jar](/assets/attachments/org.wso2.carbon.identity.sample.extension.authenticators-5.7.0.jar) file and paste inside the
+    1.  Download the [org.wso2.carbon.identity.sample.extension.authenticators-5.7.0.jar](../../assets/attachments/org.wso2.carbon.identity.sample.extension.authenticators-5.7.0.jar) file and paste inside the
         `              <IS_HOME>/repository/components/dropins             ` directory.
 
-    2.  Download the [sample-auth.war](/assets/attachments/sample-auth.war) file and paste it inside the `             <IS_HOME>/repository/deployment/server/webapps            `
+    2.  Download the [sample-auth.war](../../assets/attachments/sample-auth.war) file and paste it inside the `             <IS_HOME>/repository/deployment/server/webapps            `
```
Comment (sherenewso2) [inline en/docs/tutorials/configuring-a-service-provider-for-adaptive-authentication.md]: this .war file and the attachments folder has not been added to the repo. please check all relevant .war files and .jar files.

File: en/docs/getting-started/provisioning-architecture.md (hunk 2 above)
Comment (gomathyk) [reply en/docs/getting-started/provisioning-architecture.md]: Corrected.

File: en/docs/getting-started/single-sign-on-and-identity-federation.md (hunk 3 above)
Comment (gomathyk) [reply en/docs/getting-started/single-sign-on-and-identity-federation.md]: Corrected

**Pull request merged**